sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowedValue.java

```diff
@@ -48,7 +49,7 @@
  *
  * @param <T> the type of the value
  */
-public abstract class WindowedValue<T> {
+public abstract class WindowedValue<T>  implements Serializable {
```

Comment:
Right, Added BoundedWindow to be serializable as it is a member of of one of WindowedValue implementations.

Generic T is also a member but we can not control it